runtime/include/bpftime_logger.hpp

```diff
@@ -0,0 +1,73 @@
+#include <string>
+#include "spdlog/spdlog.h"
+#include "spdlog/cfg/env.h"
+#include "spdlog/sinks/rotating_file_sink.h"
+#include "spdlog/sinks/stdout_color_sinks.h"
+#include <cstdlib>
+#include <iostream>
+#include <filesystem>
+#include <fstream>
+
+namespace bpftime
+{
+
+inline std::string expand_user_path(const std::string &input_path)
+{
+	if (input_path.empty()) {
+		return "console";
+	}
+
+	if (input_path[0] == '~') {
+		const char *homeDir = getenv("HOME");
+		if (!homeDir) {
+			return "console";
```

Comment:
Will string "console" be parsed as a path?

Reply:
Yes, to stderr. The console log output will also be used in CI

Reply:
> Yes, to stderr. The console log output will also be used in CI

I mean, if user want to print things to a file named `console`

Reply:
That might still be print to the console...Any suggestion for handling this case?

Reply:
For example, use std::optional<string> as the return type of this function. If $HOME is not set, returning empty optional